conbench/entities/benchmark_result.py

```diff
@@ -332,12 +318,10 @@ def to_dict_for_json_api(benchmark_result):
                 "q1": to_float(benchmark_result.q1),
                 "q3": to_float(benchmark_result.q3),
                 "iqr": to_float(benchmark_result.iqr),
-                # Note(JP): it's interesting that this was added here into
-                # `stats`, that's not a nice separation between input and
-                # output.`
-                "z_score": benchmark_result.z_score,
-                "z_regression": z_regression(benchmark_result.z_score),
-                "z_improvement": z_improvement(benchmark_result.z_score),
+                # TODO: remove these.
+                "z_score": None,
+                "z_regression": False,
+                "z_improvement": False,
```

Comment:
This change makes our previous implicit behavior explicit.

Reply:
> This change makes our previous implicit behavior explicit.

I might be slow here, but why do we think we  can't we remove these yet?

Reply:
I bet we can. But I wanted to limit this PR to only the compare API for now. I had to touch this file since we can't import the `z_regression()` etc functions from `_comparator.py` anymore.

Reply:
Good choice!